data/docs/instrumentation/manual-instrumentation/java/opentelemetry-java-annotations.mdx

```diff
@@ -0,0 +1,180 @@
+---
+date: 2025-09-23
+id: java-annotations
+title: Java OpenTelemetry Instrumentation using Annotations
```

Comment:
Title and description seems inaccurate. Java otel instrumentation is being done automatically, annotations just helps to create spans automatically